README.md

```diff
@@ -73,52 +73,47 @@ Here is a list of reasons how we improved on zap with slog.
-   to log Go structures.
+1. Automatic structured logging of Go structures (including private fields)
 
-   - This was very verbose and most of the time we ended up only implementing `fmt.Stringer` and using `zap.Stringer`
-     instead.
-   - slog handles Go structures transparently, including private fields. One
-     may implement [`slog.Value`](https://godoc.org/cdr.dev/slog#Value) to override the representation,
+   - One may implement [`slog.Value`](https://godoc.org/cdr.dev/slog#Value) to override the representation,
      use struct tags to ignore or rename fields and even reuse the
      [`json.Marshal`](https://golang.org/pkg/encoding/json/#Marshal) representation
      with [`slog.JSON`](https://godoc.org/cdr.dev/slog#JSON).
+   - With zap, We found ourselves often implementing zap's
+     [ObjectMarshaler](https://godoc.org/go.uber.org/zap/zapcore#ObjectMarshaler) to log Go structures. This was
+     very verbose and most of the time we ended up only implementing `fmt.Stringer` and using `zap.Stringer`
+     instead.
 
-1. We had many helper functions for logging but we wanted the line reported to be of the parent function.
-   zap has an [API](https://godoc.org/go.uber.org/zap#AddCallerSkip) for this but it's verbose and requires
-   passing the logger around explicitly.
+1. slog takes inspriation from Go's stdlib and implements [`slog.Helper`](https://godoc.org/cdr.dev/slog#Helper) which works just like
+   [`t.Helper`](https://golang.org/pkg/testing/#T.Helper) to mark the calling function as a helper and skip it when
+   reporting location info.
 
-   - slog takes inspriation from Go's stdlib and implements [`slog.Helper`](https://godoc.org/cdr.dev/slog#Helper) which works just like
-     [`t.Helper`](https://golang.org/pkg/testing/#T.Helper)
+   - We had many helper functions for logging but we wanted the line reported to be of the parent function.
+     zap has an [API](https://godoc.org/go.uber.org/zap#AddCallerSkip) for this but it's verbose and requires
+     passing the logger around explicitly.
 
-1. We wanted tighter integration with stdlib's [`testing`](https://golang.org/pkg/testing) package.
+1. Tight integration with stdlib's [`testing`](https://golang.org/pkg/testing) package.
+   - You can configure `slogtest` to exit on any ERROR logs and it has a global stateless API
+     that takes a `*testing.T` so you do not need to create a logger first.
    - zap has [zaptest](https://godoc.org/go.uber.org/zap/zaptest) but the API surface is large and doesn't
-     integrate well. It has no support for failing on any ERROR logs nor does it have a stateless API
-     that logs directly to a [`testing.TB`](https://golang.org/pkg/testing/#TB) without creating a
-     intermediary logger.
-
-## Users
-
-If your company or project is using slog, feel free to open an issue or PR to amend this list.
-
-- [Coder](https://github.com/cdr)
```

Comment:
I liked the idea, did you remove completely for any specific reason?

Reply:
We talked on slack.